unlock-app/src/components/interface/checkout/main/Payment.tsx

```diff
@@ -64,6 +66,12 @@ export function Payment({ injectedProvider, checkoutService }: Props) {
     }
   )
 
+  const { isLoading: isCrossmintLoading, data: crossmintConfigId } =
+    useCrossmintEnabled({
+      network: lock.network,
+      lockAddress: lock.address,
+    })
+
```

Comment:
since crossmint only support single recipient, maybe add a check for when recipients.length > 1?